source/_sass/infeedl.scss

```diff
@@ -63,9 +63,16 @@ h1, h2, h3, h4, h5, h6 {
   li {
     a {
       text-transform: none;
-      color: #fff;
+      color: #F5A623;
       font-size: .9em;
       font-weight: normal;
+      border-bottom:1px solid rgba(251,192,57,.5);
+      border-bottom-style: inset;
+
+      &:hover{
+        color:#2B8EE1;
```

Comment:
Color literals like `#2B8EE1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#2B8EE1` should be written as `#2b8ee1`<br>Properties should be ordered border-bottom, color<br>Colon after property should be followed by one space